src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -3,12 +3,61 @@
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
 import core.basesyntax.model.User;
+import exceptions.RegistrationException;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_AGE = 18;
+    private static final int MAX_AGE = 150;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        checkForNull(user);
+        return (isValidValue(user)
+                && storageDao.get(user.getLogin()) == null) ? storageDao.add(user) : null;
```

Comment:
Returning null for registration is not a good idea. If email exists in DB we should also throw exception with proper message